group_vars/all.yml.sample

```diff
@@ -662,6 +663,17 @@ dummy:
 #  - { name: client.openstack, caps: { mon: "profile rbd", osd: "profile rbd pool={{ openstack_glance_pool.name }}, profile rbd pool={{ openstack_nova_pool.name }}, profile rbd pool={{ openstack_cinder_pool.name }}, profile rbd pool={{ openstack_cinder_backup_pool.name }}"}, mode: "0600" }
 
 
+#############
+# DASHBOARD #
+#############
+#dashboard_enabled: True
```

Comment:
This reminds me that we should probably make this default to `False` since dashboard won't be (fully) usable without a `ceph-grafana` node.